centreon/src/Core/Service/Application/UseCase/AddService/AddService.php

```diff
@@ -121,6 +123,10 @@ public function __invoke(AddServiceRequest $request, AddServicePresenterInterfac
             $this->assertParameters($request);
             $newServiceId = $this->createService($request);
 
+            if (isset($this->accessGroups) && $this->accessGroups !== []) {
```

Comment:
$this->accessGroups should be initialized with an empty array above line 118, it's instead of doing an isset() or you can initialize the propertie directly like `private array $accessGroups = [];`

Reply:
Corrected